bundles/org.openhab.binding.tibber/src/main/java/org/openhab/binding/tibber/internal/handler/TibberHandler.java

```diff
@@ -330,8 +345,9 @@ public void open() {
             }
```

Comment:
Optional, but now that you are at it and presumably testing the binding also: Would you consider getting rid of deprecated WebSocketClient constructor above in line 336? I believe it could be rewritten to:

```java
client = new WebSocketClient(new HttpClient(sslContextFactory));
```

Reply:
Corrected according to suggestion, and tested/confirmed ok.